x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+SCM=""
+if [[ "${PV%9999}" != "${PV}" ]] ; then # Live ebuild
+	SCM=git-r3
+	EGIT_BRANCH=master
+	EGIT_REPO_URI="https://github.com/intel/intel-vaapi-driver"
+fi
+
+AUTOTOOLS_AUTORECONF="yes"
+inherit autotools-multilib ${SCM}
+
+DESCRIPTION="HW video decode support for Intel integrated graphics"
+HOMEPAGE="https://github.com/intel/intel-vaapi-driver"
+if [[ "${PV%9999}" != "${PV}" ]] ; then # Live ebuild
+	SRC_URI=""
+else
+	SRC_URI="https://github.com/intel/intel-vaapi-driver/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/intel-vaapi-driver-${PV}"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="wayland X"
+
+RDEPEND=">=x11-libs/libva-2.0.0:=[X?,wayland?,drm,${MULTILIB_USEDEP}]
+	>=x11-libs/libdrm-2.4.52[video_cards_intel,${MULTILIB_USEDEP}]
+	wayland? ( >=media-libs/mesa-9.1.6[egl,${MULTILIB_USEDEP}]
+				>=dev-libs/wayland-1.11[${MULTILIB_USEDEP}]
```

Comment:
that's two tabs too many ;)

What about my question about upstream changes - did you check changelog and buildsystem?

Reply:
tabs depends on editor.
upstream changes are minimal and of course it works and builds

Reply:
> tabs depends on editor.
line 32 has one tab, line 33 has four tabs - that's nothing to do with your editor.

Reply:
so let's be it space

Reply:
no space either! 'two tabs too many' plus 'don't rely on visual length of your editor tab setting' equals *tab indentation only, one level after another, don't try to align more fine-grained*.

if you want to align those two dependencies, just make a linebreak after `wayland? (` and put them on entirely separate lines.

Reply:
are u happy now?! :D fixed